README.md

```diff
@@ -3,13 +3,12 @@
 Líf is the token of the Winding Tree platform.
 
 Líf is based on the ERC20 token protocol, with the option to also send information between token holders uint new methods transferData and transferDataFrom.
-Líf is also a DAO, the token holders can create proposals to change token variables and send value/data to another contracts.
 
 [![Build Status](https://travis-ci.org/windingtree/LifToken.svg?branch=master)](https://travis-ci.org/windingtree/LifToken)
 
 ## Requirements
 
-Node v7.6 or higher (versions before 7.6 do not support async/await)
+Node v7.6 or higher (versions before 7.6 do not support async/await that is used in the LifToken tests)
 
 ## Install
```

Comment:
what about a note about the optional `yarn install`? or we don't need it any longer?

Reply:
`yarn` is (not much but) a little more cumbersome than just `npm` so not requiring I think it's better. Maybe we could mention yarn as optional?

Reply:
i would love not to need anything else than `npm install`, just in my case i had to call `yarn`, so maybe we have to mention that. but again, would be great if not!

Reply:
ok, tried fresh `npm install` and `npm test` and everything worked smooth. no need for `yarn` in the documentation!

Reply:
cool, merging then! :)